gsutil.py

```diff
@@ -27,12 +27,12 @@
 # TODO: gsutil-beta: Distribute a pylint rc file.
 
 ver = sys.version_info
-if (ver.major == 2 and ver.minor < 7) or (ver.major == 3 and (ver.minor < 5 or ver.minor > 11)):
+if (vermajor == 3 and verminor < 5) or (vermajor == 4):
```

Comment:
I got this error:

```
gsutil
Traceback (most recent call last):
  File "/opt/homebrew/share/google-cloud-sdk/platform/gsutil/gsutil", line 18, in <module>
    import gsutil
  File "/opt/homebrew/share/google-cloud-sdk/platform/gsutil/gsutil.py", line 30, in <module>
    if (vermajor == 3 and verminor < 5) or (vermajor == 4):
        ^^^^^^^^
NameError: name 'vermajor' is not defined
```

This worked:
```suggestion
if (ver.major == 3 and ver.minor < 5) or (ver.major == 4):
```